### PR TITLE
BaSP-TG-94: list worked hours

### PR DESCRIPTION
**Pull request by visojulian**
Add new route to show worked hours in a certain project on a certain task.

### CODE REVIEW

File: src/Components/Employee/ListWorkedHours/index.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+      project: timesheet.project.name
+    };
+  });
+
+  if (timesheetsIsLoading) {
+    return <Spinner isLoading={true} />;
+  }
+
+  if (timesheetsError) {
+    console.log('error');
```
Comment (baudraccoverona) [inline src/Components/Employee/ListWorkedHours/index.js:38]: 🧹

File: src/routes/employees.js [hunk 2]
```diff
@@ -2,13 +2,15 @@ import React, { lazy } from 'react';
...
 const EmployeesForm = lazy(() => import('Components/Employees/Form'));
+const ListWorkedHours = lazy(() => import('Components/Employee/ListWorkedHours'));
 
 const employees = () => {
   return (
     <Switch>
       <Route exact path="/employees" component={Employees} />
       <Route exact path="/employees/form" component={EmployeesForm} />
       <Route path="/employees/form/:id" component={EmployeesForm} />
+      <Route path="/employees/list-hours/:id" component={ListWorkedHours} />
```
Comment (lmalvasia) [inline src/routes/employees.js:13]:
```suggestion
      <Route path="/employees/timesheets/:id" component={TimeSheets} />
```

File: src/routes/employees.js [hunk 3]
```diff
@@ -2,13 +2,15 @@ import React, { lazy } from 'react';
 import { Route, Switch } from 'react-router-dom/cjs/react-router-dom';
 const Employees = lazy(() => import('Components/Employees'));
 const EmployeesForm = lazy(() => import('Components/Employees/Form'));
+const ListWorkedHours = lazy(() => import('Components/Employee/ListWorkedHours'));
```
Comment (lmalvasia) [inline src/routes/employees.js:5]:
```suggestion
const TimeSheets = lazy(() => import('Components/Employee/TimeSheets'));
```

File: src/Components/Employee/ListWorkedHours/index.js [hunk 4]
```diff
@@ -0,0 +1,75 @@
+import React, { useEffect, useState } from 'react';
+import { useHistory, useParams } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import { getTimesheets } from 'redux/timeSheets/thunks';
+import { Button, Modal, Spinner, Table } from 'Components/Shared';
+import styles from './list.module.css';
+
+const ListWorkedHours = () => {
```
Comment (lmalvasia) [inline src/Components/Employee/ListWorkedHours/index.js:8]:
```suggestion
const TimeSheets = () => {
```
(rename also the folder)

File: src/Components/Employee/ListWorkedHours/index.js [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+        variant="secondary"
+        onClick={() => {
+          history.goBack();
+        }}
+      />
+    </div>
+  );
+};
+
+export default ListWorkedHours;
```
Comment (lmalvasia) [inline src/Components/Employee/ListWorkedHours/index.js:75]:
```suggestion
export default TimeSheets;
```

**Review by lmalvasia — APPROVED** [review]

**Pull request merged**